src/main/java/discord4j/discordjson/json/OnboardingEditPromptData.java

```diff
@@ -0,0 +1,35 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.Id;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+import java.util.List;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableOnboardingEditPromptData.class)
+@JsonDeserialize(as = ImmutableOnboardingEditPromptData.class)
+public interface OnboardingEditPromptData {
```

Comment:
Maybe can be better make a "base" (ex: OnboardingPrompt, OnboardingPromptFields) interface with all the data and make OnboardingPromptData extends that? mostly for reduce the redundant code here.